pkg/cli/admin/release/new.go

```diff
@@ -23,7 +23,7 @@ import (
 	"github.com/blang/semver"
 	"github.com/docker/docker/pkg/archive"
 	"github.com/ghodss/yaml"
-	digest "github.com/opencontainers/go-digest"
+	"github.com/opencontainers/go-digest"
```

Comment:
Packages where the final directory name is different from their package name are annoying.  The explicit `digest` here is a way to underline that this package will be referred to as `digest` in the rest of the file, so folks don't have to wonder what `go-digest` picked internally as their name.  But :shrug:, I'll survive without the explicit local hint; it's not like folks wondering where `digest` is defined will have many search-hits in this file to sift through, so they'll probably be able to figure it out.